src/Catty.xcodeproj/project.pbxproj

```diff
@@ -808,10 +808,9 @@
 		6F659875243E8B9B00BD6B8C /* StoreProjectDownloader.downloadProject.fail.request.json in Resources */ = {isa = PBXBuildFile; fileRef = 6F659874243E8B9B00BD6B8C /* StoreProjectDownloader.downloadProject.fail.request.json */; };
 		6F87B80D2441300A00D79A2C /* StoreProjectDownloader.downloadProject.success.json in Resources */ = {isa = PBXBuildFile; fileRef = 6F87B80C2441300A00D79A2C /* StoreProjectDownloader.downloadProject.success.json */; };
 		6FAAD8B6244CAFDA00142FA7 /* SpeakBrickWithoutCommentedOut_0991.xml in Resources */ = {isa = PBXBuildFile; fileRef = 6FAAD8B5244CAFDA00142FA7 /* SpeakBrickWithoutCommentedOut_0991.xml */; };
+		6FAAD9252452B85A00142FA7 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6FAAD9242452B85A00142FA7 /* AppDelegate.swift */; };
 		6FDC3CB82439423D0033F252 /* RegularExpressionFunction.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6FDC3CB52439423D0033F252 /* RegularExpressionFunction.swift */; };
 		6FDC3CBA243942980033F252 /* RegularExpressionFunctionTest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6FDC3CB9243942980033F252 /* RegularExpressionFunctionTest.swift */; };
-		6F659875243E8B9B00BD6B8C /* StoreProjectDownloader.downloadProject.fail.request.json in Resources */ = {isa = PBXBuildFile; fileRef = 6F659874243E8B9B00BD6B8C /* StoreProjectDownloader.downloadProject.fail.request.json */; };
```

Comment:
Could you please give me a short info why these files are deleted. Were they added to the wrong target (i.e. Catty and CattyTests)?

Reply:
I have no idea how did this happen, moreover they were not even added to the wrong target still the build and tests are also passing. I have undone the changes.

Reply:
I have noticed that whenever I remove main.m, AppDelegate.m and AppDelegate.h those reference also gets removed. I don't know why this is happening.

please correct me if I am doing something wrong.

Reply:
Maybe there was something wrong with merging those changes in the other pr. However, looks right since they are still in the project and tests are passing. Will have a closer look at it.